### ISSUE TITLE
PDF Issue In the Web Version

### ISSUE THREAD

**Issue by keshavgbpecdelhi** [microsoft/Web-Dev-For-Beginners#507]
**Describe the bug**
It's doing well with README.md as their PDF is accessing using [pdf/readme.pdf](https://github.com/microsoft/Web-Dev-For-Beginners/blob/main/pdf/readme.pdf) but in [Website](https://microsoft.github.io/Web-Dev-For-Beginners) it is linking to [https://microsoft.github.io/Web-Dev-For-Beginners/#/pdf/readme.pdf](https://microsoft.github.io/Web-Dev-For-Beginners/#/pdf/readme.pdf)  which is giving 404-Not found. We need to edit it to [https://microsoft.github.io/Web-Dev-For-Beginners/pdf/readme.pdf](https://microsoft.github.io/Web-Dev-For-Beginners/pdf/readme.pdf) as it's the real link. 

TLDR; the '#' is causing problem beacuse of **docsify**

**Additional context**
I tried to search how to fix it but didn't found anything, help it explaining too, Thanks.

**Comment by jlooper** [comment]
I think we need to add the full path. If you want to PR I will push

**Comment by keshavgbpecdelhi** [comment]
Yes, let me do 🗽
